Allow multiple fields in optionkey

mciszczon opened the pull request:
This PR allows multiple fields values in `optionkey`, as mentioned in #9. It also introduces new blueprint option of `optiondivider` that allows one to set a custom divider between these fields.

mciszczon committed:
Multiple option keys

mciszczon committed:
Fix README

calebgrove commented:
Thanks! I’m out of the office right now, but I’ll check this over as soon as I can.

mciszczon committed:
Change isEmpty to !=''
Changed isEmpty() to != '' comparison. Aditionally, the default separator was changed to ` — ` to reflect the README.

mciszczon committed:
Fix README no. 2

mciszczon committed:
Merge branch 'master' of https://github.com/mciszczon/select-a-structure

mciszczon commented:
b8432b0 fixes a bug, where I tested string values with `isEmpty()`, thus getting an error. Sorry for the additional commits, had some inconsistencies on my local copy and that's what you get.

mciszczon committed:
Optionkey and optiondivider options added

mciszczon committed:
Update README

mciszczon committed:
Update README with optionid
